tests/python/test_bit_array_vectorization.py

```diff
@@ -32,7 +32,7 @@ def init():
 
     @ti.kernel
     def assign_vectorized():
-        get_runtime().prog.current_ast_builder().bit_vectorize(32)
+        get_runtime().prog.current_ast_builder().bit_vectorize()
```

Comment:
OOC: is this API user-facing? can we do `ti.loop_config(bit_vectorize=True)` instead? :P

Reply:
Yep. This is planned in the upcoming PR :-)